src/app/_helpers/interceptors/buildings.interceptor.ts

```diff
@@ -13,21 +13,20 @@ export class BuildingsInterceptor implements HttpInterceptor {
   intercept(req: HttpRequest<any>, next: HttpHandler): Observable<HttpEvent<any>> {
     const buildingsMock = new BuildingResponseMock();
     const buildings = buildingsMock.buildings;
-
-    if (req.url.endsWith('/game/buildings')) {
-      return utilities.sendResponse(buildings , 200);
+    if (req.url.endsWith(environment.getBuildings)) {

```

Comment:
@HonzaDudek can we extract the endpoint names to something under the environment? sg like `environment.backend.getBuildings`

Reply:
probably not even put in the environments, but something that is environment independent